packages/async-operations/src/async_operations.ts

```diff
@@ -95,6 +104,53 @@ export const startECSTask = async ({
   }).promise();
 };
 
+export const createAsyncOperation = async (
+  params: {
+    createObject: ApiAsyncOperation,
```

Comment:
Curious to hear why you named this `createObject` 🤔

Reply:
Gives it some delineation between the fact that this the object represents the async operation that we are going to create, but hasn't been created yet until this function returns